Documentation/Troubleshooting/ceph-csi-common-issues.md

```diff
@@ -425,40 +425,66 @@ kubectl -n rook-ceph delete pod my-app-69cd495f9b-nl6hf --grace-period 0 --force
 
 After the force delete, wait for a timeout of about 8-10 minutes. If the pod still not in the running state, continue with the next section to blocklist the node.
 
-### Blocklisting a node
+### Blocklisting a node or holder Pod
 
 To shorten the timeout, you can mark the node as "blocklisted" from the [Rook toolbox](ceph-toolbox.md) so Rook can safely failover the pod sooner.
 
 If the Ceph version is at least Pacific(v16.2.x), run the following command:
 
+If csi host networking is enabled, we can blocklist the Node IP. If not, we need to blocklist the holder pod IP running on the faulty node.
+
+#### Getting holder Pod IP
+
+```console
+kubectl get po -o wide -nrook-ceph
```

Comment:
Is there a label we can use so they only get the holders? And a couple other minor suggestions on the formatting, I know it works either way, but suggesting this for readability.
```suggestion
kubectl get pod -o wide -n rook-ceph -l <label>
```